vshard/storage/init.lua

```diff
@@ -2795,6 +2798,12 @@ local function rebalancer_download_states()
             replicaset, 'vshard.storage.rebalancer_request_state', {},
```

Comment:
Now you'll print:

```
<...>: Some buckets in replicaset %s are not active! Will retry rebalancing every %s s
<...>: Error during downloading rebalancer states: %s'
<...>: Some buckets are not active, retry rebalancing later
```

That's not good, let's instead return nil, replicaset.id from the `rebalancer_download_states` and just print a more meaningful message from the `rebalancer_service_f`